### ISSUE TITLE
Error: Cannot find module './abc.js'

### ISSUE THREAD

**Issue by horaceho** [lukehaas/RunJS#262]
There are two `.js` files in a "Hello" folder:
```
Hello % ls -a
.	..	abc.js	test.js
```
```
Hello % cat abc.js 
var Abc = function () {
    function hello() {
        return 'hello world!';
    }

    return {
        hello: function () {
            return hello();
        },
    };
};

if (typeof exports !== "undefined") exports.Abc = Abc;

if (typeof define !== "undefined")
    define(function () {
        return Abc;
    });
```
```
Hello % cat test.js 
const { Abc } = require('./abc.js');
const abc = new Abc();

console.log(abc.hello());
```
```
Hello % node test.js 
hello world!
```
The above runs fine in node. However, when I put the same `test.js` code in RunJS and set working folder to "Hello", got this error:
![Screenshot 2021-06-24 at 09 15 49](https://user-images.githubusercontent.com/6657/123187476-d52f5d00-d4cc-11eb-99a6-b4d5eecee50d.png)

**Comment by lukehaas** [comment]
I'll get this fixed as soon as I can 👍

**Comment by lukehaas** [comment]
This is resolved in version 2.0.1
However, the path for the require needs to be absolute rather than relative, like this `require('abc.js')`

**Comment by horaceho** [comment]
Wonderful!
